azurerm/resource_arm_virtual_machine.go

```diff
@@ -296,6 +296,7 @@ func resourceArmVirtualMachine() *schema.Resource {
 								string(compute.StorageAccountTypesPremiumLRS),
 								string(compute.StorageAccountTypesStandardLRS),
 								string(compute.StorageAccountTypesStandardSSDLRS),
+								string(compute.StorageAccountTypesUltraSSDLRS),
```

Comment:
Should this also be added to the os disk block too?

Reply:
"Ultra SSD can only be used as data disks. " 
https://docs.microsoft.com/en-us/azure/virtual-machines/linux/disks-types